### PR TITLE
Add offboarding checklist

### PR DESCRIPTION
**Pull request by dadlerj**

### CODE REVIEW

**Comment by dadlerj** [comment]
This should be merged after https://github.com/sourcegraph/about/pull/560

**Comment by dadlerj** [comment]
@mercadon bump on this

File: handbook/people-ops/index.md [hunk 1]
```diff
@@ -1,19 +1,28 @@
 # People Ops
 
+## Team
+- [Teammate philosophy](teammate_philosophy.md)
+- [List of team members](../../company/team/index.md)
```
Comment (ryan-blunden) [inline handbook/people-ops/index.md]:
```suggestion
- [Team members](../../company/team/index.md)
```

**Comment by sqs** [comment]
@dadlerj Do you still intend to merge this PR? If not, please close it. I'll close it in a couple days if I haven't heard back (I'm just trying to clean up our PR list).

**Pull request closed without merging**